packages/node_modules/@webex/widget-recents/src/enhancers/setup.js

```diff
@@ -148,11 +148,13 @@ function getInitialSpaces(props) {
        * and decrypt individually to give a good initial user experience.
        */
       props.fetchSpacesEncrypted(sparkInstance, {
-        conversationsLimit: 25
+        conversationsLimit: props.spaceLoadCount
```

Comment:
does this have a default value?

Reply:
Yes, 25.

Reply:
https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/widget-recents/src/container.js#L87